### PR TITLE
[ACC-1242] Update to npm@7

### PR DESCRIPTION
**Pull request by aasanchez**
Scope:
Upgrade to npm7

Changes:
* Need to remote 2 jobs in circleci, related to [Old OpenSSH version on CircleCI](https://github.com/financial-times/next/wiki/npm-7-Migration-Guide#old-openssh-version-on-circleci)
* Install python in the node container, this is a solution to [this](https://app.circleci.com/pipelines/github/Financial-Times/n-syndication/829/workflows/40c3d937-e4cb-403f-b253-55eea1820939/jobs/3276) fail in the build 
* Refactor in the README.md

### CODE REVIEW

**Comment by aasanchez** [comment]
I need information about the release process for this package, I can see to be able to deploy this, is need to set a new version.

My question is, maybe this will bring breaking changes to other projects, we should jump to version `6.x.x` or keep in `5.x.x`

**Review by binaryberry — COMMENTED** [review]
Looks good to me as far as I can tell! And congrats on the 100th n-syndication PR. It will require a release of n-syndication - please get someone to show you how we release if you haven't done it at the FT so far.

File: package.json [hunk 1]
```diff
@@ -36,21 +37,23 @@
     "n-ui-foundations": "github:financial-times/n-ui-foundations#nobower",
     "next-session-client": "^3.0.1",
     "o-buttons": "npm:@financial-times/o-buttons@^6.0.2",
+    "o-icons": "npm:@financial-times/o-icons@^6.3.0",
```
Comment (polkwc) [inline package.json:40]: **Question**; Is this correct? To move from 7 to 6.3 ?

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "babel-preset-react": "^6.11.1",
     "bower": "^1.8.8",
     "bower-resolve-webpack-plugin": "^1.0.3",
+    "check-engine": "^1.10.1",
```
Comment (polkwc) [inline package.json:23]: **Question**; don't you have to add the package.lock.json too ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -67,15 +70,14 @@ jobs:
...
+      - run:
+          name: Install python
+          command: sudo apt-get update && sudo apt-get install python -qq
       - run:
           name: Install project dependencies
           command: make install
-      - run:
-          name: shared-helper / npm-install-peer-deps
-          command: .circleci/shared-helpers/helper-npm-install-peer-deps
-      - run:
```
Comment (polkwc) [inline .circleci/config.yml:76]: Question; it is correct to remove this job? (https://github.com/Financial-Times/session-decoder-js/blob/49bd53b8896b7cc5efd9c65a2ca651f348b25505/.circleci/config.yml#L78)

File: package.json (hunk 1 above)
Comment (aasanchez) [reply package.json:40]: This was an automatic change, I didn't touch this... must be something related to dependencies

File: package.json (hunk 2 above)
Comment (aasanchez) [reply package.json:23]: Becuase is ignored in .gitignore

File: .circleci/config.yml (hunk 3 above)
Comment (aasanchez) [reply .circleci/config.yml:76]: Yep, this is a know problem in the npm migration, is documented here https://github.com/financial-times/next/wiki/npm-7-Migration-Guide#old-openssh-version-on-circleci and was apply like here https://github.com/Financial-Times/next-ammit-api/pull/796/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L74

**Pull request merged**